tasks/mesos.yml

```diff
@@ -35,6 +40,22 @@
   notify:
     - Restart mesos-slave
 
+- name: Mesos Slave attributes file
+  template: src=attributes-mesos-slave.j2 dest=/etc/mesos-slave/attributes
+  when:
```

Comment:
Correct. If that's a little too opaque I can just use a regular old `and` here instead. I don't think the feature itself is documented, but you can see it here:

https://github.com/ansible/ansible/blob/stable-1.9/lib/ansible/utils/__init__.py#L263-L267

Works in 2.x as well.